lib/glueby/contract/payment.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+module Glueby
+  module Contract
+    # This class can send TPC between wallets.
+    #
+    # Examples:
+    #
+    # sender = Glueby::Wallet.load("wallet_id")
+    # reciever = Glueby::Wallet.load("wallet_id")
+    #                      or
+    #            Glueby::Wallet.create
+    #
+    # Balance of sender and reciever before send
+    # sender.internal_wallet.balance
+    # => 100_000(tapyrus)
+    # reciever.internal_wallet.balance
+    # => 0(tapyrus)
+    #
+    # Send
+    # Payment.transfer(sender_wallet: sender, receiver_wallet: reciever, amount: 10_000)
+    # sender.internal_wallet.balance
+    # => 90_000
+    # reciever.internal_wallet.balance
+    # => 10_000
+    #
+    class Payment
+      include Glueby::Contract::TxBuilder
+      extend Glueby::Contract::TxBuilder
+
+      class << self
+        def transfer(sender_wallet:, receiver_wallet:, amount:, fee_provider: FixedFeeProvider.new)
```

Comment:
I think the name of parameters `sender_wallet` and `receiver_wallet` can be replaced with `sender` and `receiver`

Reply:
OK, I rename `sender_wallet` and `receiver_wallet` to `sender` and `receiver`